### ISSUE TITLE
False positive bad-python3-import for cProfile module

### ISSUE THREAD

**Issue by dphoyes** [PyCQA/pylint#1612]
### Steps to reproduce
Test file:
```
import cProfile
```
Run pylint:
```
pylint --py3k test.py 
```

### Current behavior
```
************* Module test
W:  2, 0: Module moved in Python 3 (bad-python3-import)
```

### Expected behavior
No warning. cProfile is a valid Python3 module: https://docs.python.org/3.6/library/profile.html

### pylint --version output
```
pylint 1.7.2, 
astroid 1.5.3
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609]
```

**Comment by PCManticore** [comment]
Right, thanks!
